style-sample.css

```diff
@@ -0,0 +1,161 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.app-sample {    
+    height: 896px;
+    width: 1440px;
+    left: 0px;
+    top: 1680px;    
```

Comment:
Applying `top` `bottom` `right` or `left` will be applied only with a positioned element. But as this is the main app container, it shouldn't be positioned.

Also here the `height` shouldn't be hard-coded. It should be something like the width relative t the viewport. For example `100vh`

Reply:
Also, this main container should be aligned horizontally as it's not using the total width of the screen. You can use flexbox to align it.